tests/handlers/test_e2e_keys.py

```diff
@@ -1602,3 +1602,50 @@ def test_query_local_devices_appservice(self) -> None:
                 }
             },
         )
+
+    def test_check_cross_signing_setup(self) -> None:
+        # First check what happens with no master key.
+        alice = "@alice:test"
+        exists, replaceable_without_uia = self.get_success(
+            self.handler.check_cross_signing_setup(alice)
+        )
+        self.assertIs(exists, False)
+        self.assertIs(replaceable_without_uia, False)
```

Comment:
[Clippy would have said](https://rust-lang.github.io/rust-clippy/master/index.html#/bool_assert_comparison)
```suggestion
        self.assert(not exists)
        self.assert(not replaceable_without_uia)
```
but this is probably more readable? 🤷

Reply:
This is paranoia on my part, because e.g. `self.assertTrue(expr)` checks than an expression is truthy, not literally `True`. C.f. https://docs.python.org/3/library/unittest.html?highlight=unittest#unittest.TestCase.assertTrue